batch_transfer.js

```diff
@@ -0,0 +1,21 @@
+const DHTT = artifacts.require('./DHTT.sol');
+
+const fs = require('fs');
+const filename = 'addresses.txt'
+
+const reward = 1000;
+const chunk = 5;
+
+module.exports = function(callback) {
+    var addresses = fs.readFileSync(filename, 'utf8').toString().split('\n');
+
+    var dhtt;
+    DHTT.deployed().then(function(instance) {
+        dhtt = instance;
+        for (i = 0, j = addresses.length; i < j; i += chunk) {
+            batch = addresses.slice(i, i + chunk);
+            console.log('Next batch')
+            dhtt.batchTransfer(reward, batch);
+        }
```

Comment:
what if an error occurs? we may add `.catch()`

Reply:
Nothing bad happens if error occurs. It just stops working.
What would you do in this `.catch()`? How could you handle any errors here?